cpp/.gitignore

```diff
@@ -4,3 +4,8 @@ word_source*
 *.o
 sentence_splitter
 sentence_generator
+CMakeCache.txt
+CMakeFiles/
+build/
+Makefile
+cmake_install.cmake
```

Comment:
Please add a readme to this cpp that says you need a few things. like CMake and that we migrated away from scons.